integrated_channels/sap_success_factors/client.py

```diff
@@ -30,7 +32,9 @@ class SAPSuccessFactorsAPIClient(IntegratedChannelApiClient):  # pylint: disable
      completion status endpoints.
     """
 
-    SESSION_TIMEOUT = 5
+    SESSION_TIMEOUT = getattr(settings, "ENTERPRISE_SAPSF_SESSION_TIMEOUT", 5)
+    MAX_RETRIES = getattr(settings, "ENTERPRISE_SAPSF_MAX_RETRIES", 3)
+    BACKOFF_FACTOR = getattr(settings, "ENTERPRISE_SAPSF_BACKOFF_FACTOR", 1)
```

Comment:
What factors did we consider to decide values for these settings?

Reply:
SESSION_TIMEOUT was already there and I mirrored the rest of the 2 values from `degreed2` implementation.